IBPSA/Examples/package.mo

```diff
@@ -0,0 +1,11 @@
+within IBPSA;
+package Examples "Collection of models that illustrate model use and test models"
+  extends Modelica.Icons.ExamplesPackage;
+annotation (preferredView="info", Documentation(info="<html>
+<p>
+This package contains examples for the use of models that can be found in
+<a href=\"modelica://IBPSA\">
+IBPSA</a>.
+</p>
+</html>"));
+end Examples;
```

Comment:
Do we plan to have further library-wide examples? I think the Tutorial could be on one level with the UserGuide.

Reply:
Other libraries (IDEAS and Buildings) have already `Examples/Tutorials` so it would fit in best there. That would also allow other libraries to add more library-wide examples or tutorials.

Reply:
That makes sense!